testing/compile.ts

```diff
@@ -76,6 +82,9 @@ async function browserifyBundle({
     debug: true,
     standalone: 'exports',
   });
+  for (const module of ModulesToDisableBrowserifyPolyfill) {
```

Comment:
Any thoughts on how we can actually recognize one of these at compile time and warn the user?

Reply:
I'm not sure it's needed, or even desirable. For other Node built-ins without a polyfill (ex: `dns`), Browserify just silently ignores it. This change would treat `vm` as if there was no polyfill, so silently ignoring it seems fine.

Additionally, I doubt that Pack developers will ever intentionally import `vm`, but that it will come along with some transitive dependency they know nothing about and can't easily action, so the warning here could cause more confusion then provide help.